homeassistant/components/mochad.py

```diff
@@ -13,16 +13,16 @@
 from homeassistant.const import (
     EVENT_HOMEASSISTANT_START, EVENT_HOMEASSISTANT_STOP)
 from homeassistant.const import (CONF_HOST, CONF_PORT)
+from homeassistant.components import mqtt
 
-REQUIREMENTS = ['pymochad==0.2.0']
+DEPENDENCIES = ['mqtt']
```

Comment:
Is the mqtt feature optional? If it is optional we shouldn't depend on the mqtt component here, but instead set up the component if the option is true.

Reply:
Do you have any thought on how to ensure the mqtt is initialized before mochad without DEPENDENCIES?

Reply:
It doesn't need to be initialized before mochad. Look at how the mysensors gateway module sets up the mqtt component and accesses it.

Reply:
Ok. The way I wrapped mqtt publish in mochad is the same as mysensors. I removed the dependency to mqtt. It's a lazy optional dependency now.